IntegrationExample/Resources/Localizations/en-GB.lproj/Localizable.strings

```diff
@@ -0,0 +1,7 @@
+/* 
+  Localizable.strings
+  IntegrationExample
+
+  Created by Hozefa Indorewala on 17.08.23.
```

Comment:
how are they created by me and in August? 😅

Reply:
I think it uses the Base Localizable.strings file. Do I need to change them?

Reply:
no i was just wondering 😄